rust/pact_verifier_cli/build.rs

```diff
@@ -0,0 +1,41 @@
+use std::env;
```

Comment:
Note that this file was completely Cargo culted from the other package. I think the crate name of `pact-verifier-ffi` is probably better, but thought there might be a convention to follow.